Lib/test/test_embed.py

```diff
@@ -7,7 +7,9 @@
 import re
 import subprocess
 import sys
-
+import platform
```

Comment:
Please put the import in the sorted order of other imports. And add 1 or 2 lines between imports and the constant.

Reply:
Done. Thanks for your time.